backend/src/graphql/resolvers/images/sanitizeRelationshipTypes.ts

```diff
@@ -0,0 +1,7 @@
+export const sanitizeRelationshipType = (relationshipType: string) => {
+  // Cypher query language does not allow to parameterize relationship types
+  // See: https://github.com/neo4j/neo4j/issues/340
+  if (!['HERO_IMAGE', 'AVATAR_IMAGE'].includes(relationshipType)) {
```

Comment:
probably not your problem, but there are solutions to this problem
- For create with apoc: https://github.com/neo4j/neo4j/issues/340#issuecomment-793219411
- For query with normal cypher: https://github.com/neo4j/neo4j/issues/340#issuecomment-356265721

Reply:
Yes, this code was here before. Maybe I can add this as a separate PR.

Reply:
> Apologies for the intrusion. I found this PR via the open issues linked above, and I wanted to let you know that Neo4j now fully supports dynamic labels and types as of versions 2025.03 and 5.26.4 LTS, released in March of this year.
> 
> You can read more about them [here](https://neo4j.com/blog/developer/cypher-dynamism/), or read the documentation [here](https://neo4j.com/docs/cypher-manual/current/clauses/match/#dynamic-match).

That's great to hear! Unfortunately we're stuck with ecosystem and dependencies in nodejs. It will take some effort before we'll be able to upgrade to v5.

Biggest pain: https://github.com/neo4j-graphql/neo4j-graphql-js

But thank you for letting us know!